.travis.yml

```diff
@@ -1,63 +1,47 @@
 language: python
-sudo: required
-# if the https://travis-ci.org/ODM2/ODM2PythonAPI/requests ever says: missing config
-# validate at: http://lint.travis-ci.org/
-python:
-#  - "2.6"
-  - "2.7"
-#  - "3.2"
-#  - "3.3"
-#  - "3.4"
-#  - "3.5"
-#  - "3.5-dev" # 3.5 development branch
-#  - "nightly" # currently points to 3.6-dev
-# command to install dependencies
-cache:
-  directories:
-    - $HOME/.cache/pip/wheels
-    - $HOME/virtualenv/python2.7.9
-addons:
-  apt:
-    packages:
-      - python-lxml
-      - cmake
-      - unixodbc
-      - unixodbc-dev
-      - odbcinst1debian2
-      - odbcinst
-      - freetds-dev
-      - freetds-bin
-      - tdsodbc
-      - libc6
-      - e2fsprogs
-      - mysql-client
-      - libproj-dev
-      - libgeos-dev
-      - libspatialite-dev
-#services:
```

Comment:
I removed commented out code. If that is a placeholder for something we need to open an issue and fill it with information on what needs to be done.